doc/ModulePriority.md

```diff
@@ -0,0 +1,146 @@
+# How to configure priorities for modules
+
+By default, IoT Edge does not impose an ordering in the sequence in which modules are started, updated or stopped. Edge Agent by default is the first module that gets started and based on the edge deployment specification, it figures out which modules need to be started, updated or stopped and executes those operations in a non-deterministic order.
+
+The processing order of modules can be controlled by specifying the value of a module-specific property called `priority` in the IoT Edge deployment. Modules that have been assigned a higher priority will be processed before modules that have been assigned a lower priority.
+
+**It must be noted that this feature is currently only supported for the in-memory database store of Edge Hub i.e., when the environment variable `UsePersistentStorage` is set to `false`. The work to support this feature for the disk-based database store is currently in progress.**
```

Comment:
Is this note a carry-over from the other two articles? It doesn't seem relevant to module priority.

Reply:
You're right, I somehow missed removing this.